packages/UI/src/components/organisms/Navigation/NotificationPopover/NotificationPopover.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from 'react';
+import { Popover, PopoverContent, PopoverTrigger } from '../../../Popover';
+import { Bell, Settings } from 'lucide-react';
+import NotificationItem from '../../../NotificationItem/NotificationItem';
+import { useQuery } from '@tanstack/react-query';
+import { agent } from '@devlaunchers/utility';
+
+export default function NotificationPopover() {
+  const [notifications, setNotifications] = React.useState([]);
+  const newNotificationsCount = notifications.reduce((count, x) => {
+    return x.attributes.readDateTime ? count : count + 1;
+  }, 0);
+
+  function getAllNotifications() {
+    return agent.Notifications.get({
```

Comment:
Do we need to pass this object? `agent.Notifications.get` calls `requests.get`, so method should already be set to GET.